Sources/Testing/Testing.docc/MigratingFromXCTest.md

```diff
@@ -655,15 +679,12 @@ of issues:
     ```swift
     // After
     struct FoodTruckTests {
-      @Test func grillWorks() async {
```

Comment:
Did you mean to preserve the `@Test func` on this one like you did on the other snippets?

Reply:
Oops! Thanks.